src/Oracle/Oracle.Autorest/test/Remove-AzOracleOracleSubscription.Tests.ps1

```diff
@@ -0,0 +1,21 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Remove-AzOracleOracleSubscription'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Remove-AzOracleOracleSubscription.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Remove-AzOracleOracleSubscription' {
```

Comment:
Please remove this file if the cmdlet is removed.

Reply:
Updated